packages/kit/src/views/Earn/pages/EarnProtocolDetails/index.tsx

```
@@ -0,0 +1,1094 @@
+import { useActiveAccount } from '@onekeyhq/kit/src/states/jotai/contexts/accountSelector';
+import { EJotaiContextStoreNames } from '@onekeyhq/kit-bg/src/states/jotai/atoms';
+import { OneKeyLocalError } from '@onekeyhq/shared/src/errors';
+import { ETranslations } from '@onekeyhq/shared/src/locale';
+import type { ITabEarnParamList } from '@onekeyhq/shared/src/routes';
+import {
+  EModalReceiveRoutes,
+  EModalRoutes,
+  EModalStakingRoutes,
+  ETabEarnRoutes,
+  ETabRoutes,
+} from '@onekeyhq/shared/src/routes';
```

ESLint

[error] Import "ETabEarnRoutes" is only used as types.

(@typescript-eslint/consistent-type-imports)